modules/core/shared/src/main/scala/lucuma/core/model/ProposalClass.scala

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.syntax.all._
+import eu.timepit.refined.cats._
+import io.circe.Decoder
+import io.circe.generic.semiauto._
+import io.circe.refined._
+import lucuma.core.math.units.IntPercent
+import monocle.macros.GenPrism
+import monocle.{ Focus, Lens, Optional, Prism }
+import org.typelevel.cats.time.instances.duration._
+
+import java.time.Duration
+
+sealed trait ProposalClass { val minPercentTime: IntPercent }
```

Comment:
My understanding is that it is the minimum amount of the requested time for the semester that the PI requires for this proposal. I guess if they can't get that, they won't be able to do science?

Reply:
Ok, I see.  So we'll track the time request here in the proposal class?  Is this where the actual time asked for goes?  Or is that even an option?  Maybe we just add up the included observations (somehow accounting for OR groups) and do the request with that?

Reply:
Yeah. I think we somehow base the request time on the included observations. The exact calculation probably hasn't been determined... For this initial pass I was just going to use a value from the planned time summary - although I don't know if it should be PI time or Execution time. 

The discussion in Shortcut specified that we will apply these percentages to `band 1 & 2` and `band 3`. But first we need to have bands. For now, `band 3` will always be zero, but Andy wanted to leave it in the UI so people can see it will eventually exist.

Reply:
The "minimum time" is specified in hours in the PIT but here I also suggested using a percentage so that it doesn't change when the total changes.   Todd:  I think this is a percentage of the Execution Time  (PI + partner).